packages/admin/admin/src/dataGrid/CrudMoreActionsMenu.tsx

```diff
@@ -0,0 +1,192 @@
+import { MoreVertical } from "@comet/admin-icons";
+import {
+    Button,
+    Chip,
+    ChipProps,
+    Divider,
+    DividerProps,
+    ListItemIcon,
+    ListItemText,
+    Menu,
+    MenuItem,
+    MenuList,
+    MenuListProps,
+    MenuProps,
+    Typography,
+    TypographyProps,
+} from "@mui/material";
+import { Maybe } from "graphql/jsutils/Maybe";
+import * as React from "react";
+import { PropsWithChildren } from "react";
+import { FormattedMessage } from "react-intl";
+
+function CrudMoreActionsDivider(props: DividerProps) {
+    return <Divider sx={{ margin: "8px 10px", borderColor: (theme) => theme.palette.grey[50] }} {...props} />;
+}
+
+interface CrudMoreActionsGroupProps {
+    groupTitle: React.ReactNode;
+    menuListProps?: MenuListProps;
+    typographyProps?: React.ComponentProps<typeof Typography>;
+}
+
+function CrudMoreActionsGroup({ groupTitle, children, menuListProps, typographyProps }: PropsWithChildren<CrudMoreActionsGroupProps>) {
+    return (
+        <>
+            <Typography variant="subtitle2" color={(theme) => theme.palette.grey[500]} fontWeight="bold" pt="20px" px="15px" {...typographyProps}>
+                {groupTitle}
+            </Typography>
+            <MenuList {...menuListProps}>{children}</MenuList>
+        </>
+    );
+}
+
+export interface ActionItem extends React.ComponentProps<typeof MenuItem> {
+    type: "action";
+    label: React.ReactNode;
+    startAdornment?: React.ReactNode;
+}
+
+export interface DividerItem extends React.ComponentProps<typeof Divider> {
+    type: "divider";
+}
+
+type CrudMoreActionsItem = ActionItem | DividerItem;
+
+export interface CrudMoreActionsMenuProps {
+    selectionSize?: number;
+    buttonProps?: React.ComponentProps<typeof Button>;
+    menuProps?: Partial<MenuProps>;
+    chipProps?: Partial<ChipProps>;
+    groupTypographyProps?: Partial<TypographyProps>;
```

Comment:
This behaves differently than all other admin components now. We use MUI's slot system and should also do this here. See this guide: https://docs.comet-dxp.com/docs/comet-core-development/creating-customizable-admin-components

Reply:
As mentioned here, I don't think that is necessary here: https://github.com/vivid-planet/comet/pull/2115/files#r1715276460